be/src/olap/rowset/segment_v2/column_reader.cpp

```diff
@@ -272,6 +290,7 @@ FileColumnIterator::~FileColumnIterator() {
 }
 
 Status FileColumnIterator::seek_to_first() {
+    SCOPED_RAW_TIMER(&_opts.stats->block_seek_ns);
```

Comment:
I think you compute seek time redundant.
We only record seek time in SegmentIterator, not in ColumnIterator

Reply:
ok